pkg/controller/registry/resolver/resolver.go

```diff
@@ -172,6 +173,23 @@ func (r *SatResolver) SolveOperators(namespaces []string, csvs []*v1alpha1.Clust
 	return operators, nil
 }
 
+func (r *SatResolver) toBundleInstallable(entry *cache.Entry) (*BundleInstallable, error) {
```

Comment:
I'm a little confused about why we're adding a `toBundleInstallable`, when we have a `getBundleInstallable` [here](https://github.com/operator-framework/operator-lifecycle-manager/pull/2523/files#R327)

Could we not have applied the system constraints in `getBundleInstallable`? Am I mis-reading this? Or is this to help keep it clean for the downstream code to call upon?

Reply:
I think this is ok. It fells like `getBundleInstallables` is very busy doing lots of different things - it also has two sources of entries (the stack and the filter queries it launches against the cache). Plus there's a difference in scope as well - `getBundleInstallables` gets a collection of BundleInstallables given the namespace, etc. `toBundleInstallable` converts an entry into a bundle installable.